ludolph/bot.py

```diff
@@ -609,14 +610,24 @@ def _room_config(self):
             logger.error('Could not configure MUC room member list. Error was: %s (condition=%s, etype=%s)',
                          e.text, e.condition, e.etype)
 
+    def _sleekxmpp_fix_jid(self, jid):
+        """
+        SleekXMPP in version 1.3.1 returned JID object but in version 1.3.2 returned string. Backward compatible fix!
```

Comment:
The comment is misleading - after I read it I thought that Sleek has stopped using the JID object.
However, it seems that Sleek is not returning the JID only in some cases (related to MUC?)...
Looks like the MUC plugin (xep_0045.py) is using `getStanzaValues()` in `handle_groupchat_presence` and `get_stanza_values` got changed in https://github.com/fritzy/SleekXMPP/commit/79f3c1ac8f1aa0b099958e824dc53c17daf9849f#diff-c12f981bbed099442fe509b80649eec3